kodilla-testing/build/tmp/jar/MANIFEST.MF

```diff
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
```

Comment:
lokalizacje 
/build
oraz 
/out 
sa do wyrzucenia z repo. To sa twoje lokalne kompilacje i nie powinno ich byc w repo

Reply:
mazes np usunac i wcommitowac usuniecie do repo

Reply:
przesłałem commita z poprawką, możesz sprawdzić czy się udało ?